middle_finger:
  joint_limits:
    max_M_joint_angle: 67
    max_P_joint_angle: 89
    max_D_joint_angle: 72